.agent/workflows/test.md

```
@@ -86,4 +86,4 @@ Apply Priority Stack to the written tests before presenting them:
 | **Simplicity** | Could fewer tests cover the same surface? Is any test redundant? |
 | **Performance** | Is the total added execution time justified by the coverage gained? |
 
-If any gate fails, refactor the tests before reporting done.
+If any gate fails, refactor the tests before reporting done.
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)